scripts/operations/configuration/python_lib/k8s.py

```diff
@@ -32,9 +32,14 @@
 import kubernetes.config
 
 from . import common
+from .types import JSONObject
 
-K8S_CONFIGURATION = None
-K8S_API_CLIENT = None
+# To help for type hinting
+V1Client = kubernetes.client.api.core_v1_api.CoreV1Api
+V1ClientConfiguration = kubernetes.client.configuration.Configuration
```

Comment:
I find these first two names a tiny bit confusing, particularly wondering why `V1Client` isn't just `CoreV1Api` instead. As for `V1ClientConfiguration`, I'm less sure but I don't think that one even has a version string. Or maybe just importing them directly instead would be easier.

Reply:
Agreed on the naming.

I did it this way (instead of just importing them) because some other modules needed type hints for these things, and I didn't want them to have to do the imports themselves and remember the full paths, requiring them all to be updated if any of them changed. But I can probably do it in a cleaner way. Let me think on it.